bgpd/bgp_lcommunity.c

```diff
@@ -354,7 +354,9 @@ enum lcommunity_token {
 	lcommunity_token_val,
 };
 
-/* Get next Large Communities token from the string. */
+/* Get next Large Communities token from the string.
+ * Assumes str is space-delimeted and describes 0 or more
+ * valid large communities */
 static const char *lcommunity_gettoken(const char *str,
 				       struct lcommunity_val *lval,
 				       enum lcommunity_token *token)
```

Comment:
Seems that this is not necessary at all after all changes `lcommunity_token`.

Reply:
You're right; there's no need to keep this `lcommunity_token` enum around because it is no longer meaningfully modified by the function

Reply:
Latest patch no longer includes `lcommunity_token` so this should be good to go now